front/components/app/blocks/Output.jsx

```diff
@@ -6,11 +6,11 @@ import {
   ChevronRightIcon,
   ChevronDownIcon,
 } from "@heroicons/react/20/solid";
-import { useEffect, useState } from "react";
+import { useState } from "react";
 
 const ENABLE_TOP_LEVEL_AUTO_EXPAND = false;
 
-function ObjectViewer({ block, value }) {
+export function ObjectViewer({ block, value }) {
```

Comment:
Wondering if it's right to export those from here vs moving them elsewhere, wdyt?

Reply:
Seems ok to me